molecule/default/molecule.yml

```diff
@@ -9,16 +11,23 @@ lint:
 platforms:
   - name: test-docker
     image: Fedora 29
-    flavor: m1.medium
+    flavor: ci.m1.small
+    groups:
+      - fedora
```

Comment:
Shouldn't these be set in the local `clouds.yaml` file? The `molecule/<scenario>/create.yml` files also don't reflect usage of the `clouds.yaml` file.

Reply:
In this case, they are being overridden here. ci.m1.medium is the default, but I don't need all those resources so I'm opting for ci.m1.small here. Group definition is not in clouds.yaml at all.

Also, once we move to using the newer version of molecule that's soon to be released (right @seandst ?!), we will change to using the molecule_openstack_ci role which incorporates that. We aren't bothering to retrofit that into every create.yml playbook.